storage_service/locations/models/arkivum.py

```diff
@@ -0,0 +1,168 @@
+# stdlib, alphabetical
+import json
+import logging
+import os
+import requests
+import subprocess
+
+# Core Django, alphabetical
+from django.conf import settings
+from django.db import models
+
+# Third party dependencies, alphabetical
+
+# This project, alphabetical
+from common import utils
+
+# This module, alphabetical
+from . import StorageException
+from location import Location
+from package import Package
+
+if settings.DEBUG:
+    VERIFY = False
+else:
+    VERIFY = True
+
+
+class Arkivum(models.Model):
+    space = models.OneToOneField('Space', to_field='uuid')
+
+    host = models.CharField(max_length=256,
+        help_text='Hostname of the Arkivum web instance. Eg. arkivum.example.com:8443')
+    # Optionally be able to rsync
+    remote_user = models.CharField(max_length=64, null=True, blank=True,
+        help_text="Optional: Username on the remote machine accessible via passwordless ssh.")
+    remote_name = models.CharField(max_length=256, null=True, blank=True,
+        help_text="Optional: Name or IP of the remote machine.")
+
+    class Meta:
+        verbose_name = "Arkivum"
+        app_label = 'locations'
+
+    ALLOWED_LOCATION_PURPOSE = [
+        Location.AIP_STORAGE,
+    ]
+
+    def browse(self, path):
+        # Support browse so that the Location select works
+        if self.remote_user and self.remote_name:
+            return self.space._browse_ssh(path, self.remote_user, self.remote_name)
```

Comment:
Arkivum exports a CIFS share that is where the files show up as local filesystem.  We'd originally discussed having that mount be either local or remote - remote_user and remote_name will be populated if the mount is remote with the ssh username & host. Alternately, we could just SSH in to the Arkivum server that provides the fake FS.